app/src/main/java/com/example/parkingsystemjava/mvp/presenter/DialogFragmentPresenter.java

```diff
@@ -0,0 +1,29 @@
+package com.example.parkingsystemjava.mvp.presenter;
+
+import com.example.parkingsystemjava.mvp.contract.DialogFragmentContract;
+import com.example.parkingsystemjava.mvp.listener.CustomListener;
+
+public class DialogFragmentPresenter implements DialogFragmentContract.PresenterContract {
+    private DialogFragmentContract.ViewContract view;
+    private CustomListener customListenerDialogFragment;
```

Comment:
please, do not put a listener as a instance variable, pass it only where you use it

Reply:
done